agent/installer/installer.go

```diff
@@ -35,8 +35,11 @@ type installer struct {
 }
 
 // getSupportedRegistry returns a registry with installers for the supported OS and K8s
-func getSupportedRegistry(ob algo.OutputBuilder) registry {
+func getSupportedRegistry(k8sVer string, ob algo.OutputBuilder) registry {
 	reg := newRegistry()
+	if k8sVer == "" {
+		k8sVer = "v1.22.3"
```

Comment:
This again is hardcoding to a particular patch version. Can we make this `v1.22.*`?